src/app/components/MediaLoader/index.tsx

```diff
@@ -175,13 +180,26 @@ type Props = {
   blocks: MediaBlock[];
   className?: string;
   embedded?: boolean;
+  forcePlaceholderMode?: Mode;
   uniqueId?: string;
 };
 
-const MediaLoader = ({ blocks, className, embedded, uniqueId }: Props) => {
+const MediaLoader = ({
+  blocks,
+  className,
+  embedded,
+  uniqueId,
+  forcePlaceholderMode,
+}: Props) => {
+  const transcriptBlock = getTranscriptBlock(blocks);
+  const hasTranscript = !!transcriptBlock;
+
   const { lang, translations } = useContext(ServiceContext);
   const { pageIdentifier } = useContext(EventTrackingContext);
   const { enabled: adsEnabled } = useToggle('ads');
+  const determinedPlaceholderMode = useDeterminePlaceholderMode(hasTranscript);
+
+  const placeholderMode = forcePlaceholderMode ?? determinedPlaceholderMode;
```

Comment:
And then use `determinedPlaceholderMode` below?
```suggestion
  const determinedPlaceholderMode = placeholderMode ?? useDeterminePlaceholderMode(hasTranscript);
```

Reply:
This one causes a lint error because it's against functional react practices to use conditional hooks, since it messes up the order in which hooks are called during the reconciliation phase.